consensus/tool/src/main.rs

```diff
@@ -35,7 +35,7 @@ pub struct Config {
     pub tool_command: ToolCommand,
 
     /// Consensus_uri's to connect to
-    #[clap(global = true, use_value_delimiter = true, env = "MC_PEERS")]
+    #[clap(global = true, use_value_delimiter = true, env = "MC_PEER")]
```

Comment:
:clown_face:  It looks like there is still one `MC_PEERS` in use, but it seems odd this went to a singular version when it can be a comma delimited sequence of urls

Reply:
Mobilecoind has `MC_PEER` for this purpose as a comma separated list, and if we change that its a breaking change to the mobilecoind api. it seems better to just be consistent with what we have